claudio-plugin/skills/gws-doc-action-extractor/tests/test_date_parsing.py

```
@@ -0,0 +1,194 @@
+
+
+def run_extractor(gws_env, doc_content, days=9999):
+    """Run extract_recent_actions.sh against mock doc content."""
+    import os
+    gws_env.set_doc_content(doc_content)
+    env = os.environ.copy()
+    env["PATH"] = f"{gws_env.tmp_path / 'bin'}:{env['PATH']}"
+    env["GWS_MOCK_LOG"] = str(gws_env.log_file)
+    env["GWS_MOCK_RESPONSES"] = str(gws_env.responses_file)
+    cmd = ["bash", str(EXTRACT_RECENT), FILE_ID, "--days", str(days)]
+    return subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)